src/storage-preview/azext_storage_preview/_transformers.py

```diff
@@ -122,5 +124,67 @@ def transform_storage_list_output(result):
     return list(result)
 
 
+# pylint: disable=unused-argument
 def transform_file_upload(result):
     return None
+
+
+def transform_metadata(result):
+    return result.metadata
+
+
+def transform_queue_stats_output(result):
+    if type(result).__name__ == 'dict':
```

Comment:
what is the type for valid result? dict?

Reply:
It can be `dict` or `str`